Documentation/monitoring.md

```diff
@@ -9,6 +9,10 @@ Each Rook cluster has some built in metrics collectors/exporters for monitoring
 
 Note that these steps work only for Kubernetes 1.7.x or higher. For Kubernetes 1.6.x or older, refer to the Rook 0.5.0 [documentation](https://github.com/rook/rook/blob/release-0.5/Documentation/monitoring.md) and use these [manifest files](https://github.com/rook/rook/tree/release-0.5/cluster/examples/kubernetes/monitoring).
 
+If you do not have a Prometheus running yet, follow the guidelines below for an easy start. If your cluster already
```

Comment:
Just a minor suggestion to combine the first paragraph in this section with the new paragraph. For example:
```
Each Rook cluster has some built in metrics collectors/exporters for monitoring with [Prometheus](https://prometheus.io/).
If you do not have Prometheus running, follow the steps below to enable monitoring of Rook. 
\If your cluster already contains a Prometheus instance, it will automatically discover Rooks scrape endpoint
using the standard `prometheus.io/scrape` and `prometheus.io/port` annotations.
```